gauge-card/gauge-card.js

```diff
@@ -106,6 +107,16 @@ class GaugeCard extends HTMLElement {
     this._config = cardConfig;
   }
 
+  _splitEntityAndAttribute(config) {
```

Comment:
Sry for the late answer, real life got in the way. 

>doing two methods
That way the spilt would be done twice, wouldnt it?
>you can save them back to config
Thats already the case as the config object is passed by reference.

>Much easier to understand
Well... No offense, but I consider my approach to be much easier to read. When reading the setConfig method, all you need to know is that entity and attribute are split, not the details of how that is done. So in my opinion, it's good to hide the implementation behind a method with a self describing name.

Reply:
None taken, I'm always up for debate. My reasoning, based on how I read code, is that it's best to get input and output instead of having the method manipulate config by reference. 

This way users would read something like:

```
config.attribute = this._splitEntityAndAttribute(config)['attribute'];
config.entity = this._splitEntityAndAttribute(config)['entity'];
````

And the `_splitEntityAndAttribute` makes the logic of fallback or split but the users know exactly what is being manipulated (without having to understand logic) by seeing what is being returned.